processing/src/main/java/io/druid/query/aggregation/HistogramAggregatorFactory.java

```diff
@@ -99,6 +99,31 @@ public AggregatorFactory getCombiningFactory()
   }
 
   @Override
+  public AggregatorFactory getMergingFactory(AggregatorFactory other) throws AggregatorFactoryNotMergeableException
```

Comment:
I don't think this aggregator can be used at ingestion time so probably throwing UnsupportedOperationException here is fine.